### DISCUSSION TITLE
April 20th, 2023 Community Call

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#19350] [Community Call]
Agenda - 
- 7.4 Preview 3
- WDAC Audit work @PaulHigin 
- OpenSSH updates - @maertendMSFT @tgauth 
- PSGet Updates, compat module merge - @anamnavi @alerickson 
- Native piping work @SeeminglyScience 
- Docs Update @sdwheeler

**Comment by ThomasNieto** [comment]
I can demo [AnyPackage](https://anypackage.dev) a PackageManagement (OneGet) replacement. 

Currently supports the following package managers:

- [Chocolatey](https://github.com/AnyPackage/AnyPackage.Chocolatey)
- [Msu](https://github.com/AnyPackage/AnyPackage.Msu)
- [PowerShellGet v3](https://github.com/AnyPackage/AnyPackage.PowerShellGet)
- [Programs](https://github.com/AnyPackage/AnyPackage.Programs)
- [Scoop](https://github.com/AnyPackage/AnyPackage.Scoop)
- [WinGet](https://github.com/AnyPackage/AnyPackage.WinGet)

**Reply by SteveL-MSFT** [reply]
Very cool!  Would be great to have you give a demo of this!

**Comment by fflaten** [comment]
Any updates on platyPS v2 and PSSA v2? Plan and ETA for new preview?

**Reply by adityapatwardhan** [reply]
We are planning to release a new preview release of platyPS v2 in the next couple of months. It would include a bunch of cmdlets like `Update-MarkdownHelp`, `Update-MarkdownHelpModule` and `New-YamlHelp` etc.

**Comment by floh96** [comment]
Could the review of https://github.com/PowerShell/PowerShell/pull/16734 be accelerated?
According to https://github.com/PowerShell/PowerShell/pull/16734#issuecomment-1028345866 the language wg needs to review the PR but the review is pending since Februar 2022.

**Reply by SeeminglyScience** [reply]
Yep! I'm making my way through it but it's a big PR. Currently my top priority review wise

**Comment by StartAutomating** [comment]
If you'd like, I could demo a few different (fairly new) modules

* [ugit](https://github.com/StartAutomating/ugit) is always fun to talk about, and lets us get object pipeline support from git
* [PSSVG](https://github.com/StartAutomating/PSSVG) is a nifty DSL for writing SVGs in PowerShell
* [obs-powershell](https://github.com/StartAutomating/obs-powershell) is a module for controlling open broadcast studio from PowerShell
* [ShowDemo](https://github.com/StartAutomating/ShowDemo) making .demo.ps1 files a thing again
* [Piecemeal](https://github.com/StartAutomating/Piecemeal) easily add extensibility to your module
* [PipeScript](https://github.com/StartAutomating/PipeScript) lets us drastically expand the PowerShell language and write over 40 other languages with PowerShell's help.

Or maybe talk about some older modules that are very useful:

* [EZOut](https://github.com/StartAutomating/EZOut) helps you generate Types/Format files in PowerShell
* [Benchpress](https://github.com/StartAutomating/Benchpress) helps you benchmark your code
* [PSDevOps](https://github.com/StartAutomating/PSDevOps) helps you work with Azure DevOps and GitHub, and create pipelines/workflows/Github actions without ever touching YAML.


If any of these sound interesting enough, please let me know which you'd prefer so I can throw some quick demos together.

**Reply by SteveL-MSFT** [reply]
Pick whichever one you want to demo and we can save the others for future calls!

**Comment by doctordns** [comment]
Sorry i missed yesterday as I am in Marrakech!  How much did you all get through?

Back next week-got time for a chat??

From my phone so apologies for the typos.
!
________________________________

**Reply by SteveL-MSFT** [reply]
Wrong forum :)
